docs/development/testguide.rst

```diff
@@ -30,20 +29,23 @@ accessed through the ``python setup.py test`` command.
 Testing Dependencies
 ********************
 
-As of Astropy 3.0, the dependencies used by the Astropy test runner are
-provided by a separate package called `pytest-astropy`_. This package provides
-the ``pytest`` dependency itself, in addition to several ``pytest`` plugins
-that are used by Astropy, and will also be of general use to other packages.
+The dependencies used by the Astropy test runner are provided by a separate
+package called `pytest-astropy`_. This package provides the ``pytest``
+dependency itself, in addition to several ``pytest`` plugins that are used by
+Astropy, and will also be of general use to other packages.
 
 Since the testing dependencies are not actually required to install or use
-Astropy, they are not included in ``install_requires`` in ``setup.py``.
-However, for technical reasons it is not currently possible to express these
-dependencies in ``tests_require`` either. Therefore, ``pytest-astropy`` is
-listed as an extra dependency using ``extras_require`` in ``setup.py``.
-Developers who want to run the test suite will need to install the testing
-package using pip::
+Astropy, they are not included in ``install_requires`` in ``setup.cfg``.
+Instead, they are listed in an ``extras_require`` section called ``test`` in
+``setup.cfg``. Developers who want to run the test suite will need to either
+install pytest-astropy directly::
 
-    > pip install pytest-astropy
+    pip install pytest-astropy
+
+or install the core package in 'editable' mode specifying the ``[test]``
+option::
+
+    pip install .[test]
```

Comment:
No `-e` here? (I don't understand `pip` much, so may be off base)

Reply:
Fixed